v2/concordium/types.proto

```diff
@@ -583,11 +596,48 @@ message InitName {
   string value = 1;
 }
 
+// Parameter to a smart contract invocation.
+message Parameter {
+  bytes value = 1;
+}
+
 // A smart contract v0 state.
 message ContractStateV0 {
   bytes value = 1;
 }
 
+// Status of a transaction known to the node.
+message TransactionStatus {
```

Comment:
Maybe we could name this `BlockItemStatus` instead?

Below there is a type called "TransactionSummaryInBlock", which is used here, but then its main field is called `BlockItemSummary` which is just a tiny bit confusing.